### PR TITLE
Cleanup jsxgenerator

### PR DESCRIPTION
**Pull request by kdzwinel**

### CODE REVIEW

File: src/docs/components/JSXGenerator.js [hunk 1]
```diff
@@ -30,13 +29,6 @@ function generateJSX(component) {
   let jsxProps = Object.keys(component.props)
     .filter(key => key !== 'children')
     .filter(key => component.props[key] !== undefined)
-    .filter(key => {
-      const componentWithoutProp = React.cloneElement(component, {[`${key}`]: undefined});
-      const codeWithProp = ReactDOMServer.renderToStaticMarkup(component);
-      const codeWithoutProp = ReactDOMServer.renderToStaticMarkup(componentWithoutProp);
-
-      return codeWithProp !== codeWithoutProp;
-    })
```
Comment (kdzwinel) [inline src/docs/components/JSXGenerator.js]: AFAIK this was moved to `DocsActiveBlock.removePropsWithDefaultValues`

Comment (anna-wro) [reply src/docs/components/JSXGenerator.js]: not all of it - we still need to compare html to not generate jsx for default values, like normal size in Label

Comment (kdzwinel) [reply src/docs/components/JSXGenerator.js]: good point, thanks for catching that!

**Comment by kdzwinel** [comment]
Phew, updated. @anna-wro PTAL

**Comment by anna-wro** [comment]
@kdzwinel you forgot to bump the version:)

**Comment by kdzwinel** [comment]
yeah, I'm lazy :) This change doesn't influence styleguide in any way. Only the docs, so it doesn't need a version bump IMO

**Pull request merged**